safekeeper/src/wal_backup_partial.rs

```diff
@@ -433,6 +459,10 @@ pub async fn main_task(
                 && flush_lsn_rx.borrow().term == seg.term
             {
                 // we have nothing to do, the last segment is already uploaded
+                info!(
```

Comment:
FYI that's probably too noisy for being an `info!` log.

Reply:
Made it debug. Though it shouldn't be, because once partial upload is done we evict almost immediately, so shouldn't add much.